ceph_deploy/hosts/debian/install.py

```diff
@@ -30,11 +30,11 @@ def install(distro, version_kind, version, adjust_repos, **kw):
         distro.packager.add_repo_gpg_key(gpg.url(key, protocol=protocol))
 
         if version_kind == 'stable':
-            url = 'http://download.ceph.com/debian-{version}/'.format(
+            url = 'https://download.ceph.com/debian-{version}/'.format(
```

Comment:
yeah, unfortunately https doesn't work on wheezy:

```
[vagrant-debian-wheezy][DEBUG ] add deb repo to /etc/apt/sources.list.d/
[vagrant-debian-wheezy][INFO  ] Running command: env DEBIAN_FRONTEND=noninteractive DEBIAN_PRIORITY=critical apt-get --assume-yes -q update
[vagrant-debian-wheezy][WARNIN] E: The method driver /usr/lib/apt/methods/https could not be found.
[vagrant-debian-wheezy][ERROR ] RuntimeError: command returned non-zero exit status: 100
[ceph_deploy][ERROR ] RuntimeError: Failed to execute command: env DEBIAN_FRONTEND=noninteractive DEBIAN_PRIORITY=critical apt-get --assume-yes -q update
```

Looks like it's not supported at all via apt.  Weird!

I tried it with http and it worked.